frontend/src/metabase/hooks/use-sequenced-content-close-handler.ts

```diff
@@ -13,7 +13,8 @@ type PopoverData = {
 export const RENDERED_POPOVERS: PopoverData[] = [];
 
 function isEventInsideElement(e: Event, el: Element) {
-  return isElement(e.target) && el.contains(e.target);
+  const target = e.composedPath()[0];
```

Comment:
Looks like `e.composedPath()` returns an array so if it's empty `target` will be `null`, should be fine then since we catch `isElement(target)` anyway.

I also checked MDN and it's baseline, so it's good to use:

![CleanShot 2568-06-23 at 17 41 09@2x](https://github.com/user-attachments/assets/1dacebfb-1f03-41fb-bc9c-fa25d3114b1b)

Reply:
That's true, even better!